manifests/04-crd.yaml

```diff
@@ -0,0 +1,14 @@
+---
```

Comment:
@ewolinetz is this the right crd that we want to use? I suspect it's not. Can you advise?

Reply:
This looks better to me, we want to move into a matching namespace pattern.
I had a slight concern that if the code tried to get/create an `Elasticsearch` api object it would have the incorrect group afterwards, but I didn't see any references to it anywhere.